bin/configs/csharp-httpclient-net9.yaml

```diff
@@ -11,3 +11,4 @@ additionalProperties:
   useOneOfDiscriminatorLookup: true
   targetFramework: net9.0
   equatable: true
+  nullableReferenceTypes: true
```

Comment:
The default for this property should be true for > .net6(?), so I don't think you need to specify `nullableReferenceTypes: true` for these.

Reply:
Yes, that is true, it is. My thought was to have one of the language samples utilizing it to ensure it is covered in the validation/test. I'm ok to remove though